src/query/service/src/servers/http/v1/load.rs

```diff
@@ -183,19 +178,13 @@ pub async fn streaming_load(
                     };
                 }
 
-                if format.to_lowercase().as_str() == "ndjson"
-                    || format.to_lowercase().as_str() == "jsoneachrow"
-                {
-                    ndjson_source_pipe_builder(context.clone(), &plan, multipart).await
-                } else {
-                    Err(poem::Error::from_string(
-                        format!(
-                            "Streaming load only supports csv format, but got {}",
-                            format
-                        ),
-                        StatusCode::BAD_REQUEST,
-                    ))
-                }
+                Err(poem::Error::from_string(
+                    format!(
+                        "Streaming load only supports csv format, but got {}",
```

Comment:
We don't support other formats after this PR?

Reply:
It's the same as previous.